utils/wasm-instrument/Cargo.toml

```
@@ -1,9 +1,14 @@
 [package]
 name = "gear-wasm-instrument"
 version = "0.1.0"
+description = "WASM instrument library for gear programs"
```

I prefer `Wasm` as they write at https://webassembly.org

> WebAssembly (abbreviated Wasm) is a binary instruction format for a stack-based virtual machine. 
```suggestion
description = "Wasm instrument library for Gear programs"
```